src/main/scala/io/citrine/lolo/bags/BaggedResult.scala

```diff
@@ -135,7 +136,7 @@ case class SinglePredictionBaggedResult(
       var tNot: Double = 0.0
       var tNotCount: Int = 0
       vecN.indices.foreach { jdx =>
-        cov = cov + (vecN(jdx) - nTot) * (treePredictions(jdx) - expected)
+        cov = cov + (vecN(jdx) - 1) * (treePredictions(jdx) - expected)
```

Comment:
This is a bug, but it has no effect on the calculation because `\sum_b (treePredictions(b) - expected) = 0`